audits/manifest/background-color.js

```diff
@@ -26,13 +26,16 @@ class ManifestBackgroundColor {
     return 'Contains background_color';
   }
 
-  static audit(inputs) {
-    let hasBackgroundColor = false;
-    const manifest = inputs.manifest;
+  static hasBackgroundColorValue(manifest) {
+    /* eslint-disable camelcase */
+    return Boolean(manifest !== undefined &&
+                   manifest.background_color !== undefined &&
```

Comment:
`manifest.background_color` will exist if `manifest` exists, its `value` will just be undefined